<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>584</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>SSC Hygro-Thermometer Datenansicht</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QCustomPlot" name="plot" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QCheckBox" name="checkBoxLegende">
             <property name="toolTip">
              <string>Plot Legende anzeigen</string>
             </property>
             <property name="text">
              <string>Legende</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxIdealHum">
             <property name="toolTip">
              <string>Setze idealen Luftfeuchtigkeitswert</string>
             </property>
             <property name="minimum">
              <number>40</number>
             </property>
             <property name="maximum">
              <number>65</number>
             </property>
             <property name="value">
              <number>50</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushFoto">
             <property name="toolTip">
              <string>Speichere ein Foto vom aktuellen Plot</string>
             </property>
             <property name="text">
              <string>Foto</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="plotUebertragenFlach">
             <property name="toolTip">
              <string>Plot mit 300px Höhe einfügen</string>
             </property>
             <property name="text">
              <string>Plot einfügen (flach)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="plotUebertragen">
             <property name="toolTip">
              <string>Plot mit 600px Höhe einfügen</string>
             </property>
             <property name="text">
              <string>Plot einfügen (hoch)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonBold">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="toolTip">
              <string>Fett</string>
             </property>
             <property name="text">
              <string>F</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonUnterstrichen">
             <property name="font">
              <font>
               <underline>true</underline>
              </font>
             </property>
             <property name="toolTip">
              <string>Unterstrichen</string>
             </property>
             <property name="text">
              <string>U</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonItalic">
             <property name="font">
              <font>
               <italic>true</italic>
              </font>
             </property>
             <property name="toolTip">
              <string>Kursiv</string>
             </property>
             <property name="text">
              <string>K</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonColor">
             <property name="toolTip">
              <string>Text Farbe</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButtonBackColor">
             <property name="toolTip">
              <string>Hintergrund Farbe</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFontComboBox" name="fontComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Schrift</string>
             </property>
             <property name="editable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboSize">
             <property name="toolTip">
              <string>Schriftgröße</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit">
           <property name="undoRedoEnabled">
            <bool>true</bool>
           </property>
           <property name="lineWrapMode">
            <enum>QTextEdit::FixedPixelWidth</enum>
           </property>
           <property name="lineWrapColumnOrWidth">
            <number>630</number>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTableView" name="tableView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1077</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Da&amp;tei</string>
    </property>
    <addaction name="actionUSB_Logger"/>
    <addaction name="action_ffne_CSV_Datei"/>
    <addaction name="actionDaten_entfernen"/>
    <addaction name="actionOffnen"/>
    <addaction name="actionSpeichern"/>
    <addaction name="actionExport"/>
   </widget>
   <widget class="QMenu" name="Hilfe">
    <property name="title">
     <string>Hi&amp;lfe</string>
    </property>
    <addaction name="action_Hilfe"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="Hilfe"/>
  </widget>
  <action name="action_ffne_CSV_Datei">
   <property name="text">
    <string>&amp;Daten aus CSV-Datei hinzufügen</string>
   </property>
  </action>
  <action name="actionDaten_entfernen">
   <property name="text">
    <string>Daten &amp;entfernen</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>E&amp;xport PDF</string>
   </property>
  </action>
  <action name="action_Hilfe">
   <property name="text">
    <string>&amp;Hilfe</string>
   </property>
   <property name="toolTip">
    <string>Hilfe</string>
   </property>
  </action>
  <action name="actionSpeichern">
   <property name="text">
    <string>&amp;Speichern</string>
   </property>
  </action>
  <action name="actionOffnen">
   <property name="text">
    <string>&amp;Öffnen</string>
   </property>
  </action>
  <action name="actionUSB_Logger">
   <property name="text">
    <string>&amp;USB-Logger</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>action_ffne_CSV_Datei</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>addDataFromFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDaten_entfernen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>clearCSVList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addDataFromFile()</slot>
  <slot>clearCSVList()</slot>
 </slots>
</ui>
